core-bundle/src/EventListener/HttpCache/StripCookiesSubscriber.php

```diff
@@ -30,8 +30,10 @@ class StripCookiesSubscriber implements EventSubscriberInterface
         '_ga',
         '_gid',
         '_gat',
+        '_dc_gtm_.+',
         'AMP_TOKEN',
         '_gac_.+',
+        '__utm[atbczv]',
```

Comment:
According to https://developers.google.com/analytics/devguides/collection/analyticsjs/cookie-usage#experiments, there is also `__utmx` and `__utmxx`, therefore we should probably use `'__utm.+'` as @Toflar has suggested.